site/src/components/form.md

```diff
@@ -215,7 +270,48 @@ If you want to skip the basic validation on an input, apply a `data-skip-validat
 				</span>
 			</label>
 		</div>
-	</div>
+	</fieldset>
+
+	<fieldset class="field">
+		<legend class="field-label">Multiple Checkbox Field label</legend>
+		<div class="field-body">
+			<label class="checkbox">
```

Comment:
An obligatory comment here, since this _seems to me at least_ to be validating that "a required checkbox is required" (always true anyway), not "one of these checkboxes in this group is required". As long as that is understood, and we know it won't break stuff, I think we're alright.

Reply:
Let me address this.  I think it is blanketed by the group label.  Ping you once that change is in.